images/macos/helpers/Tests.Helpers.psm1

```diff
@@ -1,3 +1,5 @@
+Import-Module "$PSScriptRoot/Common.Helpers.psm1"
+
 # Invokes command and validate that the exit code is 0
 function Validate-ZeroExitCode($command) {
     $result = Get-CommandResult $command
```

Comment:
Could you please remove this function? We should use `ShouldReturnZeroExitCode` instead of it and having two function could confuse